yb-voyager/src/srcdb/yugabytedb.go

```diff
@@ -831,3 +834,88 @@ func (yb *YugabyteDB) GetNonPKTables() ([]string, error) {
 	}
 	return nonPKTables, nil
 }
+
+func (yb *YugabyteDB) GetReplicationConnection() (*pgconn.PgConn, error) {
```

Comment:
Let me know if there are any major differences between this and the existing PG implementation..

Reply:
Its only the sleep that I have introduced in the dropReplicationSlot function that is different from PG